### PR TITLE
Pass base context URI when processing links

### PR DESCRIPTION
**Pull request by raducoravu**
Proposed fix for:
https://github.com/dita-ot/dita-ot/issues/3078

### CODE REVIEW

File: src/main/plugins/org.dita.base/xsl/preprocess/topicpullImpl.xsl [hunk 1]
```diff
@@ -696,7 +700,10 @@ mode="topicpull:figure-linktext" and mode="topicpull:table-linktext"
         <xsl:if test="exists($shortdesc) and not(normalize-space($shortdesc) = '')">
           <xsl:apply-templates select="." mode="topicpull:add-genshortdesc-PI"/>
           <desc class="- topic/desc ">
-            <xsl:apply-templates select="$shortdesc"/>
+            <xsl:apply-templates select="$shortdesc">
+              <!-- EXM-42721 Pass the base context element. -->
```
Comment (jelovirt) [inline src/main/plugins/org.dita.base/xsl/preprocess/topicpullImpl.xsl]: This should be removed

**Comment by jelovirt** [comment]
DCO is missing.

**Comment by jelovirt** [comment]
Could you squash the commits and use descriptive commit message, the current "Update topicpullImpl.xsl" is a bit generic.

**Comment by infotexture** [comment]
@raducoravu Does this resolve #3078, or are there aspects of that issue unaddressed by these changes?

**Comment by raducoravu** [comment]
@infotexture there are still cases in which the warning is shown, I gave more details and a sample in #3078. But the warning seems to be benign in that case so maybe we should leave this as such.

**Pull request merged**